e2e_samples/temperature_events/LoadTesting/IoTSimulator.ps1

```diff
@@ -0,0 +1,86 @@
+<#
+This script creates several Azure Container Instances that runs the IoT Simulator.
+All resources, including the resource group, will be torn down automatically 
+after the simulators finish sending the load.
+#>
+param (
+    [string]$Location = 'japaneast',
+    [Parameter(Mandatory=$true)][string]$ResourceGroup,
+    [int]$DeviceCount = 1,
+    [int]$ContainerCount = 1,
+    [int]$MessageCount = 1,
+    [int]$Interval = 1000,
+    [string]$FixPayload = '',
+    [int]$FixPayloadSize = 0,
+    [string]$Template = '{ \"deviceId\": \"$.DeviceId\", \"temp\": $.Temp, \"Ticks\": $.Ticks, \"Counter\": $.Counter, \"time\": \"$.Time\", \"engine\": \"$.Engine\", \"source\": \"$.MachineName\" }',
```

Comment:
` \"engine\": \"$.Engine\", \"source\": \"$.MachineName\"` not sure where this came from.

Reply:
this was the default template of iot simulator when I copy and pasted. I'm not using it so let me delete it. thanks for noticing :)